src/main/java/mate/academy/bookshop/dto/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.bookshop.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import mate.academy.bookshop.validation.PasswordFieldMatch;
+import org.hibernate.validator.constraints.Length;
+
+@PasswordFieldMatch(message = "{register.repeatPassword.mismatch}")
+@Data
+public class UserRegistrationRequestDto {
+    private Long id;
+    @Email(message = "{register.email.invalid}")
+    @NotBlank(message = "{register.email.invalid}")
+    @NotNull
```

Comment:
Do you need this annotation? Read what NotBlank annotation validates.

Reply:
Let's return `@NotBlank` annotation here and remove `@NonNull`.
<img width="889" alt="image" src="https://github.com/BolsunovaNataliia/book-shop/assets/140437083/dda6b00f-750e-4049-a3ad-bf8608b7866d">